code/modules/cargo/exports/weapons.dm

```diff
@@ -11,61 +11,61 @@
 	include_subtypes = TRUE
 
 /datum/export/weapon/knife
-	cost = 100
+	cost = 500
 	unit_name = "combat knife"
 	export_types = list(/obj/item/kitchen/knife/combat)
 
 
 /datum/export/weapon/taser
-	cost = 200
+	cost = 400
 	unit_name = "advanced taser"
 	export_types = list(/obj/item/gun/energy/e_gun/advtaser)
 
 /datum/export/weapon/laser
-	cost = 200
+	cost = 600
 	unit_name = "laser gun"
 	export_types = list(/obj/item/gun/energy/laser)
 
 /datum/export/weapon/disabler
-	cost = 100
+	cost = 300
 	unit_name = "disabler"
 	export_types = list(/obj/item/gun/energy/disabler)
 
 /datum/export/weapon/energy_gun
-	cost = 300
+	cost = 700
 	unit_name = "energy gun"
 	export_types = list(/obj/item/gun/energy/e_gun)
 
 /datum/export/weapon/wt550
-	cost = 300
+	cost = 800
 	unit_name = "WT-550 automatic rifle"
 	export_types = list(/obj/item/gun/ballistic/automatic/wt550)
 
 /datum/export/weapon/shotgun
-	cost = 300
+	cost = 900
```

Comment:
Shotguns were 350 pre-bounty, although bringing their price in line with other weapons seems okay.